src/allmydata/test/web/test_web.py

```diff
@@ -494,13 +505,13 @@ def failUnlessIsFooJSON(self, res):
         self.failUnlessReallyEqual(to_bytes(data[1]["ro_uri"]), self._foo_readonly_uri)
         self.failUnlessReallyEqual(to_bytes(data[1]["verify_uri"]), self._foo_verifycap)
 
-        kidnames = sorted([unicode(n) for n in data[1]["children"]])
+        kidnames = sorted([str(n) for n in data[1]["children"]])
         self.failUnlessEqual(kidnames,
                              [self._htmlname_unicode, u"bar.txt", u"baz.txt",
                               u"blockingfile", u"empty", u"n\u00fc.txt", u"quux.txt", u"sub"])
-        kids = dict( [(unicode(name),value)
+        kids = dict( [(str(name),value)
                       for (name,value)
-                      in data[1]["children"].iteritems()] )
+                      in list(data[1]["children"].items())] )
```

Comment:
The `list` seems superfluous but also harmless

Reply:
Yeah, futurize throws them in and I don't always remove them.